src/views/AboutView.vue

```diff
@@ -1,197 +0,0 @@
-<script setup>
```

Comment:
was this file moved or something?

Reply:
idk what happened here, should check with upstream